src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Product/FormTheme/specific_prices.html.twig

```diff
@@ -0,0 +1,36 @@
+{#**
```

Comment:
Ok I see you needed two form themes, that makes sense:
- one to display the list inside the product form
- one independent to display the specific price form in the modal

Reply:
At first, I thought about renaming them to avoid confusing both files,  but since they are not in the same folder it's all good 👍